### ISSUE TITLE
check Redirection 404 log

### ISSUE THREAD

**Issue by cjyabraham** [cncf/cncf.io#204]
See if there are any frequent ones that could be handled by a redirect. 

Turn off the 404 logging if all looks good.

**Comment by cjyabraham** [comment]
@thetwopct any thoughts on why [so many sites are still trying to get](https://www.cncf.io/wp/wp-admin/tools.php?page=redirection.php&sub=404s&groupby=url) `/wp-content/uploads/2017/12/icon_cncf_twitter.png`?

**Comment by thetwopct** [comment]
> @thetwopct any thoughts on why [so many sites are still trying to get](https://www.cncf.io/wp/wp-admin/tools.php?page=redirection.php&sub=404s&groupby=url) `/wp-content/uploads/2017/12/icon_cncf_twitter.png`?

I could be wrong, but I believe that image was being used in the [old OG content](https://github.com/cncf/landscape/issues/314#issuecomment-371570617), so any old tweet or embed is probably looking for that image until it picks up it has now changed. The image was the CNCF square logo without text. I've redirected it to the 512px favicon PNG so it's the same image.

There are still 404s being generated for new blog posts.

For example:

https://www.cncf.io/blog/2020/09/15/17-cncf-interns-graduate-from-summer-of-code-gsoc-2020/ - 283 hits.

but the actual published articles URL is:

https://www.cncf.io/blog/2020/09/17/16-cncf-interns-graduate-from-summer-of-code-gsoc-2020/

Different date, different permalink. The wrong link is even on [Google](https://www.google.com/search?client=safari&rls=en&sxsrf=ALeKk01_SJnI3WlN_lJmbDvuvkfXjbIvRg%3A1600444653906&ei=7dhkX6vfNoKckgXaw7agCQ&q=https%3A%2F%2Fwww.cncf.io%2Fblog%2F2020%2F09%2F15%2F17-cncf-interns-graduate-from-summer-of-code-gsoc-2020%2F&oq=https%3A%2F%2Fwww.cncf.io%2Fblog%2F2020%2F09%2F15%2F17-cncf-interns-graduate-from-summer-of-code-gsoc-2020%2F&gs_lcp=CgZwc3ktYWIQAzoECAAQR1CdmwFYnZsBYN6cAWgAcAF4AIABfIgBfJIBAzAuMZgBAKABAqABAaoBB2d3cy13aXrIAQjAAQE&sclient=psy-ab&ved=0ahUKEwjridD4iPPrAhUCjqQKHdqhDZQQ4dUDCAw&uact=5).

One of the sources of the traffic could be this [tweet](https://twitter.com/thenewstack/status/1306832076038176768). I've checked kubeweekly and the CNCF newsletter to see if the links were there, but they weren't.

So my initial thought is that it was put live, then updated, and the editor didn't realise the permalink had changed, and no automatic redirect was setup.

**Comment by cjyabraham** [comment]
Thanks for setting up the redirect for the twitter url.

I have setup the redirect for the blog post and let Ihor (the author) know.  I have also activated the Redirection permalink monitoring.  I had originally turned this off as, previously, I had found it created a lot of noise (redirects that weren't needed).  It should be easy, however, to see which redirects it is creating as they will all go in the ["Modified Posts" group](https://www.cncf.io/wp/wp-admin/tools.php?page=redirection.php&filterby%5Bgroup%5D=2), so we can keep an eye on it.
